index.js

```diff
@@ -138,11 +138,11 @@ sudokuSolver.solve = function (options) {
     solver(problem);
 
     return {
-        toString: function(style) {
+        toString: function () {
             // We just return the stringified solution (styled or not)
             return output (_solution, style, true)
```

Comment:
@ffflorian It's actually defined on line 71 which is fine! Totally agree! :+1: 

I guess you have to update the docs as well.

 - https://github.com/radubogdan/node-sudoku-solver#solve-optionstostringstyle
 - https://github.com/radubogdan/node-sudoku-solver#solve-optionsprintsolutionstyle

:memo: